### DISCUSSION TITLE
How to configure CoucheDB in Budibase

### DISCUSSION THREAD

**Discussion by stockarea** [Budibase/budibase#12236] [Help] [answered]
I wanted to know if we can change CoucheDB to some other database which Budibase can use for its purpose like MYSQL or POSTGRES.

And also if I want to use a different coucheDB server running on some other machine, how can configure that in Budibase?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @stockarea,

I believe Michael answered a question similar to this [here](https://github.com/Budibase/budibase/discussions/12286#discussioncomment-7491486).
